Project.toml

```
@@ -13,6 +13,7 @@ VTKBase = "4004b06d-e244-455f-a6ce-a5f9919cc534"
 
 [compat]
 CodecZlib = "0.7"
+Downloads = "1"
```

```suggestion
Downloads = "1.6"
```
It should have the same version number as the Julia entry here - see https://discourse.julialang.org/t/psa-compat-requirements-in-the-general-registry-are-changing/104958